_data/hosting_providers.json

```diff
@@ -1619,6 +1619,16 @@
     "reviewed": "",
     "note": ""
   },
+  {
+    "name": "Seravo",
+    "link": "https://seravo.com",
+    "category": "full",
+    "tutorial": "https://help.seravo.com/article/24-how-do-i-enable-the-https-on-our-website",
```

Comment:
It looks like the user has to manually specify that they want https. What happens if the user puts in an http address?

Reply:
HTTPS is mandatory for sites at Seravo. All domains have automatically Let's Encrypt certificates and they are enabled by default. There is nothing the users need to do, unless they have first done something stupid, like manually changed stuff in their database to have http instead https.

Maybe I should just remove the tutorial link? There is nothing customers normally need to do, HTTPS is always on and mandatory.

Reply:
Ah, I see, thank you for clarifying! We only show either the "announcement" or the "tutorial", so yes, removing the tutorial link is the best call.